Vagrantfile

```diff
@@ -16,10 +17,9 @@ end
 
 ensure_plugins(vconfig.fetch('vagrant_plugins')) if vconfig.fetch('vagrant_install_plugins')
 
-wordpress_sites = load_wordpress_sites
-site_hosts = hosts(wordpress_sites)
+trellis_config = Trellis::Config.new(root_path: ANSIBLE_PATH)
 
-Vagrant.require_version '>= 1.8.5'
+Vagrant.require_version '>= 1.9.6'
```

Comment:
Any reason for this?

Reply:
Because of `dig`. It needs Ruby 2.3

Reply:
`dig` is awesome but only used in one method. Can we do without it for now?

I'd rather bump this separately. It does involve a little discussion since triggers and landrush plugins won't work :( (annoying since they are dead plugins).

Reply:
reverted